src/Clients/CaradhrasDataApiClient.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Idez\Caradhras\Clients;
+
+use Carbon\Carbon;
+
+class CaradhrasDataApiClient extends BaseApiClient
+{
+    public const API_PREFIX = 'data';
+
+    public function requestExtraction(string $service, Carbon $from, Carbon $to, string $accountId, string $compress = 'zip')
+    {
+        return $this->apiClient()->get("/v1/transactions", [
+            'service' => $service,
+            'from' => $from->format('Y-m-d'),
+            'to' => $to->format('Y-m-d'),
```

Comment:
We can use the `toDateString()` instead of a custom format.

```suggestion
            'from' => $from->toDateString(),
            'to' => $to->toDateString(),
```